specification.markdown

```diff
@@ -34,7 +34,7 @@ This document defines specific terms in order to define the format and applicati
 
 `EBML Schema`: A standardized definition for the structure of an `EBML Document Type`.
 
-`EBML Document`: A datastream comprised of only two components, an `EBML Header` and an `EBML Body`.
+`EBML Document`: A datastream comprised of only two components, an `EBML Header` and an `EBML Body`, plus any optional `Global Elements` that MAY be permitted at the Root Level (for example, the `Void Element`).
```

Comment:
IMO Global Elements are part of the EBML Body.

We may have a loophole though, as I'm not sure they are allowed in the EBML Header. Void and CRC-32 seem like they could both be used there.

Maybe there's a distinction between the Global Elements defined in the EBML spec and the Global Elements defined in the format (let's say Matroska). The latter seem tied to the Body (defined by the Doctype in the Header). It seems they should be counted as inside the Body. The former I'm not sure. We may add to the Void/CRC definitions that they can be found in the `EBML Header` and the `EBML Body`.

Reply:
So with the one word change proposed below it think we don't need this line at all. The Global Elements at the root are part of the `EBML Body`.

Then we need extra text saying that Void and CRC-32 can be found *inside* the `EBML Header`, but not at the root.

Reply:
The proposed text change is:
> EBML allows some special Elements to be found within more than one parent in an EBML Document or optionally at the Root Level of an EBML Body.

In the end we don't need extra text because the EBML Header is part of the EBML Document, so Void/CRC-32 can be in a parent element of the EBML Header. They can only be found at the root in the Body (but then CRC-32 requires a parent so it cannot).